### DISCUSSION TITLE
Can share display of user in a webrtc call?

### DISCUSSION THREAD

**Discussion by dscherdi** [pion/webrtc#1807] [General] [answered]
Hello!

I am thinking of using this library on a project of mine using flutter. I plan to embbed this library using FFI Flutter provides. Before I do that, I need to know it accomplishes what I want to achieve with this project. 
Basically a webrtc call where you can share the screen of the desktop. So wondering if this is possible with this library in all platforms (Windows/Linux/Mac/Mobile)? If not what is needed to provide that functionality?

Desclaimer:
I'm new to WebRtc, so sorry for lack of better knowledge.

Best Regards

**Comment by Sean-Der** [comment, marked as answer]
Hey @dscherdi 

Sorry for missing your question. This library just handles the WebRTC transport. You could do capture with https://github.com/pion/mediadevices

I think you might see better support across all platforms with https://github.com/flutter-webrtc/flutter-webrtc
